src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -180,17 +179,45 @@ public void jobLogic () {
         }
     }
 
+    /**
+     * Once the job is complete, notify subscribers and provide feedback on the job creation status.
+     */
     @Override
-    public void jobFinished () {
+    public void jobFinished() {
+        if (!status.error) status.completeSuccessfully("New version saved.");
+        // Send notification to those subscribed to feed version updates.
+        NotifyUsersForSubscriptionJob.createNotification(
+            "feed-updated",
+            feedSource.id,
+            getNotificationMessage(feedSource.name, status, feedVersion));
+
+    }
+
+    /**
+     * Create notification message based on the feed version validation result.
+     */
+    public static String getNotificationMessage(String feedSourceName, Status status, FeedVersion feedVersion) {
+        StringBuilder message = new StringBuilder();
         if (!status.error) {
-            // Note: storing a new feed version in database is handled at completion of the ValidateFeedJob subtask.
-            status.completeSuccessfully("New version saved.");
+            message.append(String.format("New feed version created for %s between %s - %s. ",
+                feedSourceName,
+                feedVersion.validationResult.firstCalendarDate,
+                feedVersion.validationResult.lastCalendarDate));
+            if (feedVersion.validationResult.errorCount > 0) {
+                message.append(String.format("During validation, we found %s issue(s)",
+                    feedVersion.validationResult.errorCount));
+            } else {
+                message.append("The validation check found no issues with this new dataset!");
+            }
         } else {
             // Processing did not complete. Depending on which sub-task this occurred in,
             // there may or may not have been a successful load/validation of the feed.
             String errorReason = status.exceptionType != null ? String.format("error due to %s", status.exceptionType) : "unknown error";
             LOG.warn("Error processing version {} because of {}.", feedVersion.id, errorReason);
```

Comment:
I would prefer this log statement happen in the jobFinished method body.

Reply:
Updated.